block_cascade/decorators.py

```diff
@@ -148,6 +152,11 @@ def remote_func(*args, **kwargs):
         # so that it can be sent to the remote executor with its parameters
         packed_func = wrapped_partial(func, *args, **kwargs)
 
+        # if running a flow locally ignore the remote resource, even if specified
+        # necessary for running a @remote decorated task in a local flow
+        if not via_cloud and no_resource_on_local:
```

Comment:
nit: add a log that resource is being ignored

Reply:
Thanks for the review.  Logging added!